module_cheatsheets/Intro_to_R_Tidyverse_cheatsheet.md

```diff
@@ -0,0 +1,82 @@
+| Library/Package                    | Piece of code        | What it's called                | What it does                                                                                                      |                                                                                 
+|------------------------------------|----------------------|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------|
+| Base `R`          | [`<-`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/assignOps.html)                    | Assignment operator   | Assigns a name to something in the R environment                                                                                                                                                  |
+| Base `R`          | [`c()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/c)                     | Concatenate           | Combines values into a vector or list                                                                                                                                                             |
+| Base `R`          | [`rm(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/remove)              | Remove                | Removes object(s) `x` from your environment                                                                                                                                                         |
+| Base `R`          | [`==, <=, >=, !=`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/Comparison.html)       | Relational Operators  | These are binary operators which allow for the comparison of values in an object.                                                                                                                 |
+| Base `R`          | [`str(x)`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/str)               | Object Structure      | Gets a summary of the object `x` structure.                                                                                                                                                         |
+| Base `R`          | [`head(x)`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/head)             | Head                  | Returns the top 6 rows of an object in the environment by default. You can specify how many rows you want by including the `n = `argument.                                                        |
+| Base `R`          | [`tail(x)`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/head)             | Tail                  | Returns the bottom 6 rows of an object in the environment. You can specify how many rows you want by including the `n =` argument.                                                                  |
```

Comment:
```suggestion
| Base `R`          | [`tail()`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/head)             | Tail                  | Returns the bottom 6 rows of an object in the environment by default. You can specify how many rows you want by including the `n =` argument.                                                                  |
```

The addition of `by default` makes it consistent with the line above this. I would remove the `x` here and anywhere else you do not explicitly talk about it in the last column.